moped-editor/src/views/projects/projectView/NoteInput.js

```diff
@@ -0,0 +1,236 @@
+);
+
+const theme = {};
+
+// Catch any errors that occur during Lexical updates and log them
+// or throw them as needed.
+const onError = (error) => {
+  console.error(error);
+}
+
+// On change, return editor content as HTML
+const OnChangePlugin = ({ onChange }) => {
+  const [editor] = useLexicalComposerContext();
+  useEffect(() => {
+    return editor.registerUpdateListener(({ editorState }) => {
+      editorState.read(() => {
+        const htmlContent = $generateHtmlFromNodes(editor, null);
+        onChange(htmlContent);
+      });
+    });
+  });
+  return null;
+}
+
+// On successful save, clear and refocus the editor
+const OnSavePlugin = ({ noteAddSuccess }) => {
+  const [editor] = useLexicalComposerContext();
+  useEffect(() => {
+    noteAddSuccess && editor.update(() => {
+      const root = $getRoot();
+      root.clear();
+    });
+    editor.focus();
+  }, [noteAddSuccess, editor]);
+  return null;
+}
+
+// On edit, load the selected comment and refocus the editor
+const OnEditPlugin = ({ htmlContent, editingNote }) => {
+  const [editor] = useLexicalComposerContext();
+  useEffect(() => {
+    const editorState = editor.getEditorState()
+    const nodeMap = editorState._nodeMap;
+    const textArray = [...nodeMap.entries()][2];
+    const isTextInEditor = !!textArray;
+    const isTextinDatabase = !!htmlContent.replace(/<[^>]*>/g, "");
+    // If user clicks edit and there is a stored note and a clear editor,
+    // populate editor with stored note
+    editingNote && !isTextInEditor && isTextinDatabase &&
+      editor.update(() => {
+        $getRoot()
+          .getChildren()
+          .forEach((n) => n.remove());
+        const parser = new DOMParser();
+        const dom = parser.parseFromString(htmlContent, 'text/html');
+        const nodes = $generateNodesFromDOM(editor, dom);
+        nodes.forEach((node) => {
+          $getRoot().append(node);
+        });
+      });
+    editor.focus();
+  }, [editingNote, htmlContent, editor]);
+  return null;
+}
+
+const NoteInput = ({
+  noteText,
+  setNoteText,
+  newNoteType,
+  setNewNoteType,
+  editingNoteType,
+  setEditingNoteType,
+  editingNote,
+  noteAddLoading,
+  noteAddSuccess,
+  submitNewNote,
+  submitEditNote,
+  cancelNoteEdit,
+  isStatusEditModal,
+}) => {
+  const classes = useStyles();
+
+  const initialConfig = {
+    namespace: 'MyEditor',
+    theme,
+    nodes: [
+      LinkNode,
+    ],
+    onError,
+  };
+
+  const onChange = (htmlContent) => {
+    setNoteText(htmlContent);
+  };
+
+  return (
+    <Container>
+      <Grid container direction="column" spacing={1}>
+        <Grid item xs={12} sm={12}>
+          <Box className={classes.editorWrapper} pt={2}>
+            <LexicalComposer initialConfig={initialConfig} >
+              <RichTextPlugin
+                contentEditable={<ContentEditable className={classes.contentEditable} />}
+                placeholder={<div className={classes.placeholder}>Enter some text...</div>}
```

Comment:
I like the new placeholder text! Could you add a style to it so that it uses the same font family as the rest of the app? You can access it through the `theme` object (i think it is `theme.typography.fontFamily`?)